tensorflow/python/ops/nn_ops.py

```diff
@@ -1601,8 +1601,7 @@ def leaky_relu(features, alpha=0.2, name=None):
     features = ops.convert_to_tensor(features, name="features")
     if features.dtype.is_integer:
       features = math_ops.to_float(features)
-    alpha = ops.convert_to_tensor(alpha, dtype=features.dtype, name="alpha")
```

Comment:
This should be changed to respect our forward compatibility rules. Use the forward compatibility layer to make this change, with the date set 3 weeks in the future of the merge (so a month from now is safe): https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/compat/compat.py

Reply:
Well, I tried to use compat.forword_compatible() on my new implementation. However, another concern (maybe over considered) is that, whether somebody would use 'alpha' as a trainable variable in his/her model? Should we try to keep compatibility for this case?